prowler/lib/check/check.py

```diff
@@ -591,8 +592,7 @@ def execute_checks(
                 check_findings = execute(
                     service,
                     check_name,
-                    global_provider.type,
-                    global_provider.identity,
+                    global_provider,
```

Comment:
This is to fix a bug, since the `execute` function only accepts the `global_provider` object.